lib/identity_cache/query_api.rb

```diff
@@ -154,6 +148,25 @@ def cache_fetch_includes
       end
     end
 
+    included do |base|
+      # Make sure there is at least once after_commit callback so that _run_commit_callbacks
+      # is called, which is overridden to do an early after_commit callback
+      base.after_commit {}
```

Comment:
interesting, I wonder what kind of overhead we are incurring here? if we had 100s of models we'd be adding 100s of callbacks; while they are NOOP we probably still have allocate many objects for no reason.

This onto itself is not a reason not to ship, but would be good to remember to measure that somehow when we bump the version of the library on a reasonably large app.

Reply:
We were previously doing this twice per model, now it is done only once, so I don't think it is that significant.  I could switch to using a no-op method or re-using a proc to avoid creating a proc per-model.

Reply:
you are right, don't mind me